cockatrice/cockatrice.qrc

```diff
@@ -341,6 +341,9 @@
         <file>resources/userlevels/admin_buddy.svg</file>
         <file>resources/userlevels/admin_vip.svg</file>
         <file>resources/userlevels/admin_vip_buddy.svg</file>
-
+        
+        <file>resources/tips/tips_of_the_day.xml</file>
+        <file>resources/tips/images/context_in_hand.png</file>
```

Comment:
Ok so this image has to be removed asap. We cannot store any copyright information within the code base. Good example, just make sure to black out certain things :)

Reply:
removing this should be done via a force push, since simply deleting it would still include it in history.

Reply:
We'll be squashing anyway, so it shouldn't matter once merged

Reply:
ah, i forgot about squash on merge haha